docs/docker_container.md

```diff
@@ -82,12 +82,12 @@ as well as a release package (.tar.gz, with ovms binary and necessary libraries)
 Use command `make docker_build BASE_OS=redhat` or `make docker_build BASE_OS=centos`. OVMS with ubi base image doesn't support NCS and HDDL accelerators.
 
 Additionally you can set version of GPU driver used by the produced image. Currently following versions are available:
-- 19.41.14441
-- 20.35.17767
+- 21.49.21786 - Redhat
+- 21.48.21782 - Ubuntu
 
 Provide version from the list above as INSTALL_DRIVER_VERSION argument in make command to build image with specific version of the driver like 
-`make docker_build INSTALL_DRIVER_VERSION=19.41.14441`. 
-If not provided, version 20.35.17767 is used.
+`make docker_build INSTALL_DRIVER_VERSION=21.49.21786`.
+If not provided, version 21.49.21786 is used for Redhat os and 21.48.21782 is used for Ubuntu os.
```

Comment:
```suggestion
If not provided, version 21.49.21786 is used for Redhat and 21.48.21782 is used for Ubuntu.
```

Reply:
I think was could drop this whole section about using INSTALL_DRIVER_VERSION paramter. Only one version will be officially supported this time for a given os.